charts/incubator/misskey/templates/ConfigMap.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Configmap needs to be in tpl format, loaded in common.yaml after the init of values.